runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -683,12 +815,28 @@ public void onGameStateChanged(GameStateChanged gameStateChanged)
 	{
 		switch (gameStateChanged.getGameState())
 		{
+			case LOGGING_IN:
+				loggingIn = true;
+				break;
+			case LOADING:
+				if (!loggingIn)
+				{
+					updateTzhaarTimerOnLoading();
+				}
+				break;
 			case HOPPING:
+				loggingIn = true;
```

Comment:
Add an explicit comment indicating the fallthrough that happens here
```suggestion
				loggingIn = true;
				// fall through to LOGIN_SCREEN case
```